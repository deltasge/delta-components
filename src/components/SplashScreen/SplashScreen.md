```
const { Button } = require('@material-ui/core');

initialState = {
  loading: false,
  error: false
};
loading = () => {
  const { loading } = state
  setState({ loading: !loading, error: false })
}
error = () => {
  const { error } = state
  setState({ loading: false, error: !error })
}

<div>
  <SplashScreen logo='https://picsum.photos/200/300/?random' loading={state.loading} error={state.error && 'Message error'} />
  <br/>
  <Button variant='contained' onClick={loading}>Loading</Button>
  <Button variant='contained' onClick={error}>Error</Button>
</div>
```
