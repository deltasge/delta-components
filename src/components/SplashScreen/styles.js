export default (theme) => ({
  root: {
    flexGrow: 1,
    height: '100%',
    zIndex: 1,
    overflow: 'hidden',
    position: 'relative',
    display: 'flex'
  },
  logoContainer: {
    textAlign: 'center'
  },
  logo: {
    maxWidth: '450px',
    [theme.breakpoints.up('md')]: {
      maxWidth: '400px'
    },
    [theme.breakpoints.down('sm')]: {
      maxWidth: '300px'
    }
  },
  title: {
    fontWeight: '500'
  },
  subtitle: {
    margin: theme.spacing.unit * 2
  },
  progress: {
    marginTop: theme.spacing.unit * 4
  }
})
