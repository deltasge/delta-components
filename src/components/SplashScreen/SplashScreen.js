import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import Grid from '@material-ui/core/Grid'
import CircularProgress from '@material-ui/core/CircularProgress'
import Typography from '@material-ui/core/Typography'

import SnackbarContent from '../Snackbar/SnackbarContent'

import styles from './styles'

/**
 * Componente para simular um splash screen
 */

const SplashScreen = ({ logo, classes, loading, error, subtitle }) => (
  <Grid container direction='column' justify='center' alignItems='center' className={classes.root}>
    <Grid item>
      <div className={classes.logoContainer}>
        <img src={logo} alt='Logo' className={classes.logo} />
      </div>
      {subtitle && <Typography className={classes.subtitle} color='primary' variant='h5' align='center' gutterBottom>{subtitle}</Typography>}
    </Grid>
    <Grid item>
      { loading && <CircularProgress color='secondary' className={classes.progress} /> }
      { error && <SnackbarContent variant='error' message={error || 'Erro inesperado'} />}
    </Grid>
  </Grid>
)

SplashScreen.defaultProps = {
  loading: false
}

SplashScreen.propTypes = {
  /** Apresentar o loading */
  loading: PropTypes.bool,
  /** Mensagem de erro */
  error: PropTypes.string,
  /** Logo */
  logo: PropTypes.string.isRequired,
  /** Sub titulo */
  subtitle: PropTypes.string
}
export default withStyles(styles)(SplashScreen)
