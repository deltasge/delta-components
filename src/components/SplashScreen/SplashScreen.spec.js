/* global describe it expect */

import React from 'react'
import SplashScreen from './SplashScreen'
import { mount } from 'enzyme'

describe('<SplashScreen />', () => {
  const error = 'Message Error'
  const logo = 'https://picsum.photos/200/300/?random'
  it('should mount', () => {
    const wrapper = mount(<SplashScreen logo={logo} error={error} />)
    expect(wrapper.containsMatchingElement(<img src={logo} alt='Logo' />)).toEqual(true)
    expect(wrapper.find(`span#client-snackbar`).text()).toEqual(error)
  })
  it('matches the snapshot', () => {
    const tree = mount(<SplashScreen logo={logo} loading />)
    expect(tree).toMatchSnapshot()
  })
  it('should mount loading state', () => {
    const wrapper = mount(<SplashScreen logo={logo} loading />)
    // console.log(wrapper.html())
    expect(wrapper.find('circle').length).toEqual(1)
  })
})
