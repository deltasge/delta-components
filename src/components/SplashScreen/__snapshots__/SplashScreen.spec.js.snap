// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplashScreen /> matches the snapshot 1`] = `
<WithStyles(SplashScreen)
  loading={true}
  logo="https://picsum.photos/200/300/?random"
>
  <SplashScreen
    classes={
      Object {
        "logo": "SplashScreen-logo-3",
        "logoContainer": "SplashScreen-logoContainer-2",
        "progress": "SplashScreen-progress-6",
        "root": "SplashScreen-root-1",
        "subtitle": "SplashScreen-subtitle-5",
        "title": "SplashScreen-title-4",
      }
    }
    loading={true}
    logo="https://picsum.photos/200/300/?random"
  >
    <WithStyles(Grid)
      alignItems="center"
      className="SplashScreen-root-1"
      container={true}
      direction="column"
      justify="center"
    >
      <Grid
        alignContent="stretch"
        alignItems="center"
        className="SplashScreen-root-1"
        classes={
          Object {
            "align-content-xs-center": "MuiGrid-align-content-xs-center-19",
            "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-21",
            "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-20",
            "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-23",
            "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-22",
            "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-18",
            "align-items-xs-center": "MuiGrid-align-items-xs-center-15",
            "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-17",
            "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-16",
            "container": "MuiGrid-container-7",
            "direction-xs-column": "MuiGrid-direction-xs-column-10",
            "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-11",
            "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-12",
            "grid-lg-1": "MuiGrid-grid-lg-1-78",
            "grid-lg-10": "MuiGrid-grid-lg-10-87",
            "grid-lg-11": "MuiGrid-grid-lg-11-88",
            "grid-lg-12": "MuiGrid-grid-lg-12-89",
            "grid-lg-2": "MuiGrid-grid-lg-2-79",
            "grid-lg-3": "MuiGrid-grid-lg-3-80",
            "grid-lg-4": "MuiGrid-grid-lg-4-81",
            "grid-lg-5": "MuiGrid-grid-lg-5-82",
            "grid-lg-6": "MuiGrid-grid-lg-6-83",
            "grid-lg-7": "MuiGrid-grid-lg-7-84",
            "grid-lg-8": "MuiGrid-grid-lg-8-85",
            "grid-lg-9": "MuiGrid-grid-lg-9-86",
            "grid-lg-auto": "MuiGrid-grid-lg-auto-76",
            "grid-lg-true": "MuiGrid-grid-lg-true-77",
            "grid-md-1": "MuiGrid-grid-md-1-64",
            "grid-md-10": "MuiGrid-grid-md-10-73",
            "grid-md-11": "MuiGrid-grid-md-11-74",
            "grid-md-12": "MuiGrid-grid-md-12-75",
            "grid-md-2": "MuiGrid-grid-md-2-65",
            "grid-md-3": "MuiGrid-grid-md-3-66",
            "grid-md-4": "MuiGrid-grid-md-4-67",
            "grid-md-5": "MuiGrid-grid-md-5-68",
            "grid-md-6": "MuiGrid-grid-md-6-69",
            "grid-md-7": "MuiGrid-grid-md-7-70",
            "grid-md-8": "MuiGrid-grid-md-8-71",
            "grid-md-9": "MuiGrid-grid-md-9-72",
            "grid-md-auto": "MuiGrid-grid-md-auto-62",
            "grid-md-true": "MuiGrid-grid-md-true-63",
            "grid-sm-1": "MuiGrid-grid-sm-1-50",
            "grid-sm-10": "MuiGrid-grid-sm-10-59",
            "grid-sm-11": "MuiGrid-grid-sm-11-60",
            "grid-sm-12": "MuiGrid-grid-sm-12-61",
            "grid-sm-2": "MuiGrid-grid-sm-2-51",
            "grid-sm-3": "MuiGrid-grid-sm-3-52",
            "grid-sm-4": "MuiGrid-grid-sm-4-53",
            "grid-sm-5": "MuiGrid-grid-sm-5-54",
            "grid-sm-6": "MuiGrid-grid-sm-6-55",
            "grid-sm-7": "MuiGrid-grid-sm-7-56",
            "grid-sm-8": "MuiGrid-grid-sm-8-57",
            "grid-sm-9": "MuiGrid-grid-sm-9-58",
            "grid-sm-auto": "MuiGrid-grid-sm-auto-48",
            "grid-sm-true": "MuiGrid-grid-sm-true-49",
            "grid-xl-1": "MuiGrid-grid-xl-1-92",
            "grid-xl-10": "MuiGrid-grid-xl-10-101",
            "grid-xl-11": "MuiGrid-grid-xl-11-102",
            "grid-xl-12": "MuiGrid-grid-xl-12-103",
            "grid-xl-2": "MuiGrid-grid-xl-2-93",
            "grid-xl-3": "MuiGrid-grid-xl-3-94",
            "grid-xl-4": "MuiGrid-grid-xl-4-95",
            "grid-xl-5": "MuiGrid-grid-xl-5-96",
            "grid-xl-6": "MuiGrid-grid-xl-6-97",
            "grid-xl-7": "MuiGrid-grid-xl-7-98",
            "grid-xl-8": "MuiGrid-grid-xl-8-99",
            "grid-xl-9": "MuiGrid-grid-xl-9-100",
            "grid-xl-auto": "MuiGrid-grid-xl-auto-90",
            "grid-xl-true": "MuiGrid-grid-xl-true-91",
            "grid-xs-1": "MuiGrid-grid-xs-1-36",
            "grid-xs-10": "MuiGrid-grid-xs-10-45",
            "grid-xs-11": "MuiGrid-grid-xs-11-46",
            "grid-xs-12": "MuiGrid-grid-xs-12-47",
            "grid-xs-2": "MuiGrid-grid-xs-2-37",
            "grid-xs-3": "MuiGrid-grid-xs-3-38",
            "grid-xs-4": "MuiGrid-grid-xs-4-39",
            "grid-xs-5": "MuiGrid-grid-xs-5-40",
            "grid-xs-6": "MuiGrid-grid-xs-6-41",
            "grid-xs-7": "MuiGrid-grid-xs-7-42",
            "grid-xs-8": "MuiGrid-grid-xs-8-43",
            "grid-xs-9": "MuiGrid-grid-xs-9-44",
            "grid-xs-auto": "MuiGrid-grid-xs-auto-34",
            "grid-xs-true": "MuiGrid-grid-xs-true-35",
            "item": "MuiGrid-item-8",
            "justify-xs-center": "MuiGrid-justify-xs-center-24",
            "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-25",
            "justify-xs-space-around": "MuiGrid-justify-xs-space-around-27",
            "justify-xs-space-between": "MuiGrid-justify-xs-space-between-26",
            "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-28",
            "spacing-xs-16": "MuiGrid-spacing-xs-16-30",
            "spacing-xs-24": "MuiGrid-spacing-xs-24-31",
            "spacing-xs-32": "MuiGrid-spacing-xs-32-32",
            "spacing-xs-40": "MuiGrid-spacing-xs-40-33",
            "spacing-xs-8": "MuiGrid-spacing-xs-8-29",
            "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-13",
            "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-14",
            "zeroMinWidth": "MuiGrid-zeroMinWidth-9",
          }
        }
        component="div"
        container={true}
        direction="column"
        item={false}
        justify="center"
        lg={false}
        md={false}
        sm={false}
        spacing={0}
        wrap="wrap"
        xl={false}
        xs={false}
        zeroMinWidth={false}
      >
        <div
          className="MuiGrid-container-7 MuiGrid-direction-xs-column-10 MuiGrid-align-items-xs-center-15 MuiGrid-justify-xs-center-24 SplashScreen-root-1"
        >
          <WithStyles(Grid)
            item={true}
          >
            <Grid
              alignContent="stretch"
              alignItems="stretch"
              classes={
                Object {
                  "align-content-xs-center": "MuiGrid-align-content-xs-center-19",
                  "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-21",
                  "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-20",
                  "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-23",
                  "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-22",
                  "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-18",
                  "align-items-xs-center": "MuiGrid-align-items-xs-center-15",
                  "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-17",
                  "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-16",
                  "container": "MuiGrid-container-7",
                  "direction-xs-column": "MuiGrid-direction-xs-column-10",
                  "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-11",
                  "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-12",
                  "grid-lg-1": "MuiGrid-grid-lg-1-78",
                  "grid-lg-10": "MuiGrid-grid-lg-10-87",
                  "grid-lg-11": "MuiGrid-grid-lg-11-88",
                  "grid-lg-12": "MuiGrid-grid-lg-12-89",
                  "grid-lg-2": "MuiGrid-grid-lg-2-79",
                  "grid-lg-3": "MuiGrid-grid-lg-3-80",
                  "grid-lg-4": "MuiGrid-grid-lg-4-81",
                  "grid-lg-5": "MuiGrid-grid-lg-5-82",
                  "grid-lg-6": "MuiGrid-grid-lg-6-83",
                  "grid-lg-7": "MuiGrid-grid-lg-7-84",
                  "grid-lg-8": "MuiGrid-grid-lg-8-85",
                  "grid-lg-9": "MuiGrid-grid-lg-9-86",
                  "grid-lg-auto": "MuiGrid-grid-lg-auto-76",
                  "grid-lg-true": "MuiGrid-grid-lg-true-77",
                  "grid-md-1": "MuiGrid-grid-md-1-64",
                  "grid-md-10": "MuiGrid-grid-md-10-73",
                  "grid-md-11": "MuiGrid-grid-md-11-74",
                  "grid-md-12": "MuiGrid-grid-md-12-75",
                  "grid-md-2": "MuiGrid-grid-md-2-65",
                  "grid-md-3": "MuiGrid-grid-md-3-66",
                  "grid-md-4": "MuiGrid-grid-md-4-67",
                  "grid-md-5": "MuiGrid-grid-md-5-68",
                  "grid-md-6": "MuiGrid-grid-md-6-69",
                  "grid-md-7": "MuiGrid-grid-md-7-70",
                  "grid-md-8": "MuiGrid-grid-md-8-71",
                  "grid-md-9": "MuiGrid-grid-md-9-72",
                  "grid-md-auto": "MuiGrid-grid-md-auto-62",
                  "grid-md-true": "MuiGrid-grid-md-true-63",
                  "grid-sm-1": "MuiGrid-grid-sm-1-50",
                  "grid-sm-10": "MuiGrid-grid-sm-10-59",
                  "grid-sm-11": "MuiGrid-grid-sm-11-60",
                  "grid-sm-12": "MuiGrid-grid-sm-12-61",
                  "grid-sm-2": "MuiGrid-grid-sm-2-51",
                  "grid-sm-3": "MuiGrid-grid-sm-3-52",
                  "grid-sm-4": "MuiGrid-grid-sm-4-53",
                  "grid-sm-5": "MuiGrid-grid-sm-5-54",
                  "grid-sm-6": "MuiGrid-grid-sm-6-55",
                  "grid-sm-7": "MuiGrid-grid-sm-7-56",
                  "grid-sm-8": "MuiGrid-grid-sm-8-57",
                  "grid-sm-9": "MuiGrid-grid-sm-9-58",
                  "grid-sm-auto": "MuiGrid-grid-sm-auto-48",
                  "grid-sm-true": "MuiGrid-grid-sm-true-49",
                  "grid-xl-1": "MuiGrid-grid-xl-1-92",
                  "grid-xl-10": "MuiGrid-grid-xl-10-101",
                  "grid-xl-11": "MuiGrid-grid-xl-11-102",
                  "grid-xl-12": "MuiGrid-grid-xl-12-103",
                  "grid-xl-2": "MuiGrid-grid-xl-2-93",
                  "grid-xl-3": "MuiGrid-grid-xl-3-94",
                  "grid-xl-4": "MuiGrid-grid-xl-4-95",
                  "grid-xl-5": "MuiGrid-grid-xl-5-96",
                  "grid-xl-6": "MuiGrid-grid-xl-6-97",
                  "grid-xl-7": "MuiGrid-grid-xl-7-98",
                  "grid-xl-8": "MuiGrid-grid-xl-8-99",
                  "grid-xl-9": "MuiGrid-grid-xl-9-100",
                  "grid-xl-auto": "MuiGrid-grid-xl-auto-90",
                  "grid-xl-true": "MuiGrid-grid-xl-true-91",
                  "grid-xs-1": "MuiGrid-grid-xs-1-36",
                  "grid-xs-10": "MuiGrid-grid-xs-10-45",
                  "grid-xs-11": "MuiGrid-grid-xs-11-46",
                  "grid-xs-12": "MuiGrid-grid-xs-12-47",
                  "grid-xs-2": "MuiGrid-grid-xs-2-37",
                  "grid-xs-3": "MuiGrid-grid-xs-3-38",
                  "grid-xs-4": "MuiGrid-grid-xs-4-39",
                  "grid-xs-5": "MuiGrid-grid-xs-5-40",
                  "grid-xs-6": "MuiGrid-grid-xs-6-41",
                  "grid-xs-7": "MuiGrid-grid-xs-7-42",
                  "grid-xs-8": "MuiGrid-grid-xs-8-43",
                  "grid-xs-9": "MuiGrid-grid-xs-9-44",
                  "grid-xs-auto": "MuiGrid-grid-xs-auto-34",
                  "grid-xs-true": "MuiGrid-grid-xs-true-35",
                  "item": "MuiGrid-item-8",
                  "justify-xs-center": "MuiGrid-justify-xs-center-24",
                  "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-25",
                  "justify-xs-space-around": "MuiGrid-justify-xs-space-around-27",
                  "justify-xs-space-between": "MuiGrid-justify-xs-space-between-26",
                  "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-28",
                  "spacing-xs-16": "MuiGrid-spacing-xs-16-30",
                  "spacing-xs-24": "MuiGrid-spacing-xs-24-31",
                  "spacing-xs-32": "MuiGrid-spacing-xs-32-32",
                  "spacing-xs-40": "MuiGrid-spacing-xs-40-33",
                  "spacing-xs-8": "MuiGrid-spacing-xs-8-29",
                  "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-13",
                  "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-14",
                  "zeroMinWidth": "MuiGrid-zeroMinWidth-9",
                }
              }
              component="div"
              container={false}
              direction="row"
              item={true}
              justify="flex-start"
              lg={false}
              md={false}
              sm={false}
              spacing={0}
              wrap="wrap"
              xl={false}
              xs={false}
              zeroMinWidth={false}
            >
              <div
                className="MuiGrid-item-8"
              >
                <div
                  className="SplashScreen-logoContainer-2"
                >
                  <img
                    alt="Logo"
                    className="SplashScreen-logo-3"
                    src="https://picsum.photos/200/300/?random"
                  />
                </div>
              </div>
            </Grid>
          </WithStyles(Grid)>
          <WithStyles(Grid)
            item={true}
          >
            <Grid
              alignContent="stretch"
              alignItems="stretch"
              classes={
                Object {
                  "align-content-xs-center": "MuiGrid-align-content-xs-center-19",
                  "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-21",
                  "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-20",
                  "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-23",
                  "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-22",
                  "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-18",
                  "align-items-xs-center": "MuiGrid-align-items-xs-center-15",
                  "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-17",
                  "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-16",
                  "container": "MuiGrid-container-7",
                  "direction-xs-column": "MuiGrid-direction-xs-column-10",
                  "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-11",
                  "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-12",
                  "grid-lg-1": "MuiGrid-grid-lg-1-78",
                  "grid-lg-10": "MuiGrid-grid-lg-10-87",
                  "grid-lg-11": "MuiGrid-grid-lg-11-88",
                  "grid-lg-12": "MuiGrid-grid-lg-12-89",
                  "grid-lg-2": "MuiGrid-grid-lg-2-79",
                  "grid-lg-3": "MuiGrid-grid-lg-3-80",
                  "grid-lg-4": "MuiGrid-grid-lg-4-81",
                  "grid-lg-5": "MuiGrid-grid-lg-5-82",
                  "grid-lg-6": "MuiGrid-grid-lg-6-83",
                  "grid-lg-7": "MuiGrid-grid-lg-7-84",
                  "grid-lg-8": "MuiGrid-grid-lg-8-85",
                  "grid-lg-9": "MuiGrid-grid-lg-9-86",
                  "grid-lg-auto": "MuiGrid-grid-lg-auto-76",
                  "grid-lg-true": "MuiGrid-grid-lg-true-77",
                  "grid-md-1": "MuiGrid-grid-md-1-64",
                  "grid-md-10": "MuiGrid-grid-md-10-73",
                  "grid-md-11": "MuiGrid-grid-md-11-74",
                  "grid-md-12": "MuiGrid-grid-md-12-75",
                  "grid-md-2": "MuiGrid-grid-md-2-65",
                  "grid-md-3": "MuiGrid-grid-md-3-66",
                  "grid-md-4": "MuiGrid-grid-md-4-67",
                  "grid-md-5": "MuiGrid-grid-md-5-68",
                  "grid-md-6": "MuiGrid-grid-md-6-69",
                  "grid-md-7": "MuiGrid-grid-md-7-70",
                  "grid-md-8": "MuiGrid-grid-md-8-71",
                  "grid-md-9": "MuiGrid-grid-md-9-72",
                  "grid-md-auto": "MuiGrid-grid-md-auto-62",
                  "grid-md-true": "MuiGrid-grid-md-true-63",
                  "grid-sm-1": "MuiGrid-grid-sm-1-50",
                  "grid-sm-10": "MuiGrid-grid-sm-10-59",
                  "grid-sm-11": "MuiGrid-grid-sm-11-60",
                  "grid-sm-12": "MuiGrid-grid-sm-12-61",
                  "grid-sm-2": "MuiGrid-grid-sm-2-51",
                  "grid-sm-3": "MuiGrid-grid-sm-3-52",
                  "grid-sm-4": "MuiGrid-grid-sm-4-53",
                  "grid-sm-5": "MuiGrid-grid-sm-5-54",
                  "grid-sm-6": "MuiGrid-grid-sm-6-55",
                  "grid-sm-7": "MuiGrid-grid-sm-7-56",
                  "grid-sm-8": "MuiGrid-grid-sm-8-57",
                  "grid-sm-9": "MuiGrid-grid-sm-9-58",
                  "grid-sm-auto": "MuiGrid-grid-sm-auto-48",
                  "grid-sm-true": "MuiGrid-grid-sm-true-49",
                  "grid-xl-1": "MuiGrid-grid-xl-1-92",
                  "grid-xl-10": "MuiGrid-grid-xl-10-101",
                  "grid-xl-11": "MuiGrid-grid-xl-11-102",
                  "grid-xl-12": "MuiGrid-grid-xl-12-103",
                  "grid-xl-2": "MuiGrid-grid-xl-2-93",
                  "grid-xl-3": "MuiGrid-grid-xl-3-94",
                  "grid-xl-4": "MuiGrid-grid-xl-4-95",
                  "grid-xl-5": "MuiGrid-grid-xl-5-96",
                  "grid-xl-6": "MuiGrid-grid-xl-6-97",
                  "grid-xl-7": "MuiGrid-grid-xl-7-98",
                  "grid-xl-8": "MuiGrid-grid-xl-8-99",
                  "grid-xl-9": "MuiGrid-grid-xl-9-100",
                  "grid-xl-auto": "MuiGrid-grid-xl-auto-90",
                  "grid-xl-true": "MuiGrid-grid-xl-true-91",
                  "grid-xs-1": "MuiGrid-grid-xs-1-36",
                  "grid-xs-10": "MuiGrid-grid-xs-10-45",
                  "grid-xs-11": "MuiGrid-grid-xs-11-46",
                  "grid-xs-12": "MuiGrid-grid-xs-12-47",
                  "grid-xs-2": "MuiGrid-grid-xs-2-37",
                  "grid-xs-3": "MuiGrid-grid-xs-3-38",
                  "grid-xs-4": "MuiGrid-grid-xs-4-39",
                  "grid-xs-5": "MuiGrid-grid-xs-5-40",
                  "grid-xs-6": "MuiGrid-grid-xs-6-41",
                  "grid-xs-7": "MuiGrid-grid-xs-7-42",
                  "grid-xs-8": "MuiGrid-grid-xs-8-43",
                  "grid-xs-9": "MuiGrid-grid-xs-9-44",
                  "grid-xs-auto": "MuiGrid-grid-xs-auto-34",
                  "grid-xs-true": "MuiGrid-grid-xs-true-35",
                  "item": "MuiGrid-item-8",
                  "justify-xs-center": "MuiGrid-justify-xs-center-24",
                  "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-25",
                  "justify-xs-space-around": "MuiGrid-justify-xs-space-around-27",
                  "justify-xs-space-between": "MuiGrid-justify-xs-space-between-26",
                  "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-28",
                  "spacing-xs-16": "MuiGrid-spacing-xs-16-30",
                  "spacing-xs-24": "MuiGrid-spacing-xs-24-31",
                  "spacing-xs-32": "MuiGrid-spacing-xs-32-32",
                  "spacing-xs-40": "MuiGrid-spacing-xs-40-33",
                  "spacing-xs-8": "MuiGrid-spacing-xs-8-29",
                  "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-13",
                  "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-14",
                  "zeroMinWidth": "MuiGrid-zeroMinWidth-9",
                }
              }
              component="div"
              container={false}
              direction="row"
              item={true}
              justify="flex-start"
              lg={false}
              md={false}
              sm={false}
              spacing={0}
              wrap="wrap"
              xl={false}
              xs={false}
              zeroMinWidth={false}
            >
              <div
                className="MuiGrid-item-8"
              >
                <WithStyles(CircularProgress)
                  className="SplashScreen-progress-6"
                  color="secondary"
                >
                  <CircularProgress
                    className="SplashScreen-progress-6"
                    classes={
                      Object {
                        "circle": "MuiCircularProgress-circle-194",
                        "circleDisableShrink": "MuiCircularProgress-circleDisableShrink-197",
                        "circleIndeterminate": "MuiCircularProgress-circleIndeterminate-196",
                        "circleStatic": "MuiCircularProgress-circleStatic-195",
                        "colorPrimary": "MuiCircularProgress-colorPrimary-191",
                        "colorSecondary": "MuiCircularProgress-colorSecondary-192",
                        "indeterminate": "MuiCircularProgress-indeterminate-190",
                        "root": "MuiCircularProgress-root-188",
                        "static": "MuiCircularProgress-static-189",
                        "svg": "MuiCircularProgress-svg-193",
                      }
                    }
                    color="secondary"
                    disableShrink={false}
                    size={40}
                    thickness={3.6}
                    value={0}
                    variant="indeterminate"
                  >
                    <div
                      className="MuiCircularProgress-root-188 MuiCircularProgress-colorSecondary-192 MuiCircularProgress-indeterminate-190 SplashScreen-progress-6"
                      role="progressbar"
                      style={
                        Object {
                          "height": 40,
                          "width": 40,
                        }
                      }
                    >
                      <svg
                        className="MuiCircularProgress-svg-193"
                        viewBox="22 22 44 44"
                      >
                        <circle
                          className="MuiCircularProgress-circle-194 MuiCircularProgress-circleIndeterminate-196"
                          cx={44}
                          cy={44}
                          fill="none"
                          r={20.2}
                          strokeWidth={3.6}
                          style={Object {}}
                        />
                      </svg>
                    </div>
                  </CircularProgress>
                </WithStyles(CircularProgress)>
              </div>
            </Grid>
          </WithStyles(Grid)>
        </div>
      </Grid>
    </WithStyles(Grid)>
  </SplashScreen>
</WithStyles(SplashScreen)>
`;
